packages/jest-core/src/TestScheduler.ts

```diff
@@ -281,7 +281,7 @@ export default class TestScheduler {
     return aggregatedResults;
   }
 
-  private _partitionTests(
+  private static _partitionTests(
```

Comment:
?

Reply:
Local build seems to be failing when I remove this

Reply:
_partitionTests